### PR TITLE
Siva Reddy | Adding Drug Concepts

### PR DESCRIPTION
**Pull request by sivareddytw**
JIRA - https://bahmni.atlassian.net/browse/BS-98

- Loading drugs using initialiser module

### CODE REVIEW

File: package/docker/Dockerfile [hunk 1]
```diff
@@ -3,6 +3,7 @@ FROM alpine
 COPY openmrs /etc/bahmni_config/openmrs
 COPY openelis /etc/bahmni_config/openelis
 COPY offline /etc/bahmni_config/offline
+COPY configuration /etc/bahmni_config/masterdata/configuration
```
Comment (angshu) [inline package/docker/Dockerfile:6]: is this required? Please check with Umair and Mohan or the config team .. we didnt need it for other metadata/masterdata

File: configuration/conceptclasses/conceptclasses.csv [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Uuid,Void/Retire,Name,Description
```
Comment (angshu) [inline configuration/conceptclasses/conceptclasses.csv:1]: aren't these standard concept names? if yes, then Bahmni should define that out of box. Check with Akhil what config team is doing in this regard

File: configuration/concepts/WHODrugConcepts.csv [hunk 3]
```diff
@@ -0,0 +1,146 @@
+Uuid,Void/Retire,Same as mappings,Fully specified name:en,Short name:en,Description:en,Fully specified name:fr,Short name:fr,Description:fr,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:200
```
Comment (angshu) [inline configuration/concepts/WHODrugConcepts.csv:1]: Why WHODrugConcepts - WHO Drugs will probably have ATC codes, not SNOMED. So maybe change the file name. Also, check with Akhil if such Drug Concepts are generic and/or standard from CIEL dictionary to be bundled with Bahmni. If yes, then we probably dont have to do this using a csv but the metadata zip or sync will take care of it.

File: package/docker/Dockerfile (hunk 1 above)
Comment (sivareddytw) [reply package/docker/Dockerfile:6]: yes @angshu ... from this directory, we are copying to ${OPENMRS_APPLICATION_DATA_DIRECTORY}/configuration. From, here, iniz module will load the files.
Reference: 
https://github.com/Bahmni/openmrs-distro-bahmni/blob/master/package/docker/openmrs/bahmni_startup.sh#L14

File: configuration/concepts/WHODrugConcepts.csv (hunk 3 above)
Comment (sivareddytw) [reply configuration/concepts/WHODrugConcepts.csv]: @angshu updated the file name.

File: configuration/conceptclasses/conceptclasses.csv (hunk 2 above)
Comment (sivareddytw) [reply configuration/conceptclasses/conceptclasses.csv:1]: @angshu concepts which we are creating here are taken from lite.mybahmni.in environment (CIEL ZIP) and in this project we are using default-config (demo Bahmni). Some concepts are already present in the system but mapping codes are missing. we are updating the mappings also for the existing codes. 

Checked with Akhil, he is going to create comprehensive list of drugs for Bahmni Standard whenever he finds sometime.

File: configuration/concepts/WHODrugConcepts.csv (hunk 3 above)
Comment (sivareddytw) [reply configuration/concepts/WHODrugConcepts.csv]: @angshu Checked with Akhil, these concepts are standard from CIEL dictionary but for this project we are not using CIEL ZIP. (default-config), so creating them with corresponding mapping codes

**Review by angshu — APPROVED** [review]

**Pull request merged**